### DISCUSSION TITLE
mieru作為Tor網橋

### DISCUSSION THREAD

**Discussion by ghost** [enfein/mieru#145] [Q&A]
mieru作為Tor網橋  吸取了mieru所有特點
自己mieru部署麻煩 購買VPS不匿名! 流量貴!
如果mieru作為Tor網橋, 不需要自己部署伺服器. 無限流量 看1080p不是问题😄

分叉mieru作為Tor網橋開發合併到 Tor官方https://gitlab.torproject.org/tpo/anti-censorship/pluggable-transports

meek 死亡
Snowflake 死亡 速度100k 一分鐘必定斷線
obfs4 明文洩漏網橋ip
WebTunnel 新 明文洩漏網橋ip 或網址

**Comment by enfein** [comment]
我感觉 Tor 网桥这个项目，除非是用 cloudflare 之类的方法，否则直接封 IP 地址就可以解决问题了。

所以我对 Tor 网桥不是很期待，不如老老实实挂个自己的前置代理。前置代理最多只能看见你在用 Tor，其他一概不知道。
